bin/run.sh

```diff
@@ -166,11 +157,7 @@ case ${option} in
     echo "Info: Initiating SupportNotifications Test"
 	supportNotificationTest	| tee $SUPPORT_NOTIFICATION_LOG_PATH
     ;;
-  	-exc)
-    echo "Info: Initiating ExportClient Test"
-    exportClientTest | tee $EXPORTCLIENTLOGSPATH
-    ;;
-    -ss)
+  	-ss)
```

Comment:
@michaelestrin please restore this line 160.  You are using tab instead of spaces